src/transaction/CreateTransaction.tsx

```diff
@@ -118,6 +207,68 @@ const CreateTransaction = function CreateTransaction(props: TransactionAction) {
     }
   };
 
+  const onAddressInput = async () => {
+    const addressElem = (document.getElementById('toAddress') as HTMLInputElement);
+    const txReq : TransactionRequest = {
+      to: addressElem.value,
+    };
+
+    const isAddressValid = await SimulationSuite.isAddressValid(txReq);
+    const feedbackElem = document.getElementById('to-feedback');
+    if (!isAddressValid) {
+      addressElem.className = 'form-control is-invalid';
+      feedbackElem!.innerHTML = 'Invalid address';
+      feedbackElem!.className = 'invalid-feedback';
+    } else {
+      addressElem.className = 'form-control is-valid';
+      feedbackElem!.innerHTML = '';
+      feedbackElem!.className = 'valid-feedback';
+    }
+  };
+
+  const onAmountInput = () => {
+    const amountElem = (document.getElementById('amount') as HTMLInputElement);
+    const amountInput = amountElem.value;
+    const feedbackElem = document.getElementById('amt-feedback');
+
+    // Initial check to make sure there are no leading 0s (> 1)
+    // Ex: Preventing 00001234 --> 1234
+    const amountRegex = /^((([1-9]\d*|0)((\.\d+)?))|(\.\d+))$/;
+
+    if (amountRegex.test(amountInput)) {
+      try {
+        ethers.utils.parseEther(amountInput);
+        // const state = await getStateObj();
+        // const provider = state.provider as Provider;
+        document.getElementById('amount-in-usd')!.innerHTML = (getCurrentETHInUSD(+amountInput, currentETHValue)).toString().concat(' USD');
+        amountElem.className = 'form-control is-valid';
+        feedbackElem!.innerHTML = '';
+        feedbackElem!.className = 'valid-feedback';
+      } catch (e: any) {
+        amountElem.className = 'form-control is-invalid';
+        if (String(e).includes('fractional component exceeds')) {
+          feedbackElem!.innerHTML = 'Amount has too many decimal places';
+          feedbackElem!.className = 'invalid-feedback';
+        } else {
+          const i = String(e).indexOf('(');
+          let errMsg = String(e).substring(ERROR.length, i - 1);
+          errMsg = errMsg[0].toUpperCase().concat(errMsg.substring(1));
+          feedbackElem!.innerHTML = errMsg;
+          feedbackElem!.className = 'invalid-feedback';
+        }
+      }
+    } else {
+      amountElem.className = 'form-control is-invalid';
+      feedbackElem!.innerHTML = 'Invalid amount inputted';
```

Comment:
Same as above:

- [ ] Use `e.message`
- [ ] Use `textContent` instead of `innerHTML`